Makefile

```diff
@@ -0,0 +1,29 @@
+all: build check
+
+check: fmt vet lint ## Check your code
+
+revendor:
+	rm glide.lock
+	glide install --strip-vendor
+	glide-vc --use-lock-file --no-tests --only-code
+
+terraform:
+	hack/get-terraform.sh
+
+build: ## build installer binary
```

Comment:
Same as for the Terraform target, I'd rather this matched the name of the file being created.  In this case, it's `bin/openshift-install`.  And once you match, you'll need a `.PHONY` entry, because we aren't telling `make` about the Go dependencies.